### DISCUSSION TITLE
d-cite attributes

### DISCUSSION THREAD

**Discussion by buerviper** [alshedivat/al-folio#1305] [Q&A]
I would like to write a distill-style post with references. For some references (for instance books), I would like to indicate page numbers. I would like to avoid adding the same book dozens of times in the bibliography, just with different page numbers.

Is there any attribute I could add? Like 
```
<d-cite key="bibliography-key" page="42"></d-cite>
```
or similarly? Or is that a new feature?

**Comment by george-gca** [comment]
I might be wrong, but I believe this is currently not supported. But would be a great addition though.
